bitcoin-explained/breaking-down-taproot-activation-options.md

```diff
@@ -0,0 +1,1674 @@
+---
+title: "Breaking Down Taproot Activation Options"
+transcript_by: Sjors via rev.com
+revision_by: edilmedeiros
+media: https://youtu.be/mT0t8Jm0m5E?si=ITfvLBQkhA6KA7DG
+tags: ["bitcoin-core", "taproot", "soft-fork"]
+speakers: ["Sjors Provoost","Aaron van Wirdum"]
+categories: ["podcast"]
+date: 2020-07-25
+aliases: ['/bitcoin-magazine/bitcoin-explained/breaking-down-taproot-activation-options']

```

Comment:
no need for `aliases`. Some of those have this field because there were in a different directory (nested under `bitcoin-magazine`) and I recently moved them. This way the previous links don't break.

Reply:
I was going to ask if it would be a good idea to move stuff around as they were about the same podcast, but now I see there's a rationale to keep them like so.